DoctorWho.DB/Validation/DoctorForUpdateDtoValidator.CS

```diff
@@ -25,13 +27,17 @@ namespace DoctorWho.DB.Validation
 
 
         }
+        private DoctorForUpdateDtoValidator(DoctorWhoCoreDbContext context)
+        {
+            _context = context;
+
+        }
 
 
 
         private bool IsUniqueDoctorNumber(int? number)
         {
-            var _db = new DoctorWhoCoreDbContext();
-            var result = _db.Doctors.Where(x => x.DoctorNumber == number).SingleOrDefault();
+            var result = _context.Doctors.Where(x => x.DoctorNumber == number).SingleOrDefault();
```

Comment:
now in this validater ,I found it depend on the context ,,so I needed to pass it in away that let my program take it from startup ,I couldnt have another base class ,, so second constructor was used .
Let me know what do u think about this ,, please